dev-docs/dev-deployment.md

```diff
@@ -42,6 +42,26 @@ Set up an IP for your deployment:
 #### Troubleshooting:
 * Certbot error `[Errno 13] Permission denied: '/var/log/letsencrypt' Either run as root, or set --config-dir, --work-dir, and --logs-dir to writeable paths.` - do not run as root, but set the directories instead.
 
+## Minikube
```

Comment:
Since this is an alternative to `### Google Kubernetes`, let's move this section below that one (so before `Troubleshooting`) and add a level of `#`.

Reply:
Also, IIRC you ended up not using minikube?

Reply:
@djmitche I do use minikube.